.gitignore

```diff
@@ -0,0 +1,4 @@
+#General
```

Comment:
nit: A space separating the hash from the comment would look 'cleaner'.
```diff
-#General
+# General
```

Reply:
@sagirk Ok cool, I inserted a space and pushed it to my master.

Reply:
@erichodges Could you confirm if the push was successful? Somehow the change is not reflecting here.

Reply:
@sagirk I guess my earlier push (with the space added) was not successful. I just pushed it again and it's showing on the master branch of my fork.

Reply:
Thanks @erichodges! Looks good now.